packages/grid/src/vaadin-grid-column.js

```diff
@@ -473,11 +473,29 @@ export const ColumnBaseMixin = (superClass) =>
       }
 
       this._allCells.forEach((cell) => {
-        cell._content.style.textAlign = textAlign;
-        if (getComputedStyle(cell._content).textAlign !== textAlign) {
-          cell._content.style.textAlign = textAlignFallback;
-        }
+        this.__setCellTextAlign(cell, textAlign, textAlignFallback);
       });
+
+      let parentGroup = this.parentElement;
+      while (parentGroup.tagName === 'VAADIN-GRID-COLUMN-GROUP') {
```

Comment:
Maybe there are other ways to verify the element here? `instanceof` couldn't be used because I can't import `GridColumnGroup` since it extends `ColumnBaseMixin`.

Reply:
Would something like this be enough:
```js
if (
  this.parentElement !== this._grid &&
  this.parentElement._colSpan === 1 &&
  this.parentElement.textAlign === undefined
) {
  this.parentElement.textAlign = textAlign;
}
```